### PR TITLE
Integrate session storage with logic so all pages are linked

### PR DESCRIPTION
**Pull request by lucyrose93**
merge after #108 approved and merged

relates #108

### CODE REVIEW

**Comment by des-des** [comment]
@lucyrose93 

  1. Wait until #108 is merged
  2. merge latest master into this branch
  3. assign me

Its important that you only assign me on step 3, not before step 1 and 2

**Comment by lucyrose93** [comment]
Thanks @des-des please merge

**Review by des-des — APPROVED** [review]

**Pull request merged**
